src/tools/pkcs15-tool.c

```diff
@@ -635,6 +638,16 @@ static void print_prkey_info(const struct sc_pkcs15_object *obj)
 	printf("\tAccess Flags   : [0x%02X]", prkey->access_flags);
 	print_key_access_flags(prkey->access_flags);
 	printf("\n");
+	printf("\tAlgo_refs      : ");
+	/* zero may be valid and don't know how many were read  print at least 1*/
+	for (i = 0; i< SC_MAX_SUPPORTED_ALGORITHMS; i++) {
```

Comment:
clang-format suggested the following change:


```
-	for (i = 0; i< SC_MAX_SUPPORTED_ALGORITHMS; i++) {
+	for (i = 0; i < SC_MAX_SUPPORTED_ALGORITHMS; i++) {
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs15-tool.c -->